relay-monitors/src/lib.rs

```diff
@@ -60,6 +60,48 @@ where
     uuid.as_simple().serialize(serializer)
 }
 
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+#[serde(rename_all = "snake_case")]
+#[serde(tag = "schedule_type", content = "schedule")]
```

Comment:
Let me know if this is the way to go here, combined with the `flatten` in the field this seems to work pretty well.